release.json

```diff
@@ -10,7 +10,7 @@
         "OMNIBUS_RUBY_VERSION": "datadog-5.5.0",
         "JMXFETCH_VERSION": "0.47.9",
         "JMXFETCH_HASH": "fb5c3fc2fb42db1ee5c3a7c6187b316d79d69c50b967db0b7ffde590622d0395",
-        "MACOS_BUILD_VERSION": "master",
+        "MACOS_BUILD_VERSION": "pythyu/debug-agent-cache",
```

Comment:
I guess we will need to change it when the PR on MacOS build repo will be merged

Reply:
yea of course